manifests/init.pp

```diff
@@ -375,9 +391,7 @@
       group   => $step_tickers_group,
       mode    => $step_tickers_mode,
       content => template('ntp/step-tickers.erb'),
-      require => [ Package[$package_name_real],
```

Comment:
step-tickers needs to require the package or the package needs to be before the step-tickers file. Since the package can now optionally not be there at all, then the before/require needs to be optional as well.